onionscan.go

```diff
@@ -1,22 +1,13 @@
 package main
 
 import (
+	"github.com/s-rah/onionscan/config"
 	"github.com/s-rah/onionscan/protocol"
 	"github.com/s-rah/onionscan/report"
 	"strings"
 )
 
-type OnionScan struct {
-	TorProxyAddress string
-}
-
-func Configure(torProxyAddress string) *OnionScan {
-	onionScan := new(OnionScan)
-	onionScan.TorProxyAddress = torProxyAddress
-	return onionScan
-}
-
-func (os *OnionScan) Scan(hiddenService string) (*report.OnionScanReport, error) {
+func onionscan(onionscanConfig *config.OnionscanConfig, hiddenService string) (*report.OnionScanReport, error) {
```

Comment:
Everything else looks great, the only thing I dislike is this being an unexported function. I think we should still have an OnionScan struct with a Scan() function. More extensible and having it exportable means providing a library as well as a utility.

And then have the config be part of the OnionScan struct, #21 will add the ability to scan multiple sites, it seems unnecessary to pass this configuration to Scan for every site.